### ISSUE TITLE
Wrong "xstate/no-imperative-action" inside actions.pure()

### ISSUE THREAD

**Issue by VanTanev** [rlaffers/eslint-plugin-xstate#4]
**Describe the bug**
"xstate/no-imperative-action" is triggered with valid use of `send()`

**Actual behavior**
```typescript
import { actions, send, createMachine } from 'xstate'

export const machine = createMachine<any, any>(
    {
        on: {
            EVENT: {
                actions: actions.pure(() => {
                    return [
                      // getting eslint xstate/no-imperative-action
                      send({ type: 'BLAH' })
                    ]
                }),
            },
        },
    },
)
```
**Versions (please complete the following information):**
 - Node version :v14.15.5
 - ESLint version: eslint@7.23.0
 - eslint-plugin-xstate version: eslint-plugin-xstate@0.8.0

**Additional context**
If I use `actions.send()` it works, but I think that's because the eslint plugin doesn't actually execute in that case?

**Comment by rlaffers** [comment]
Thanks for the report. Definitely looks like a bug.

**Comment by rlaffers** [comment]
It is detected as a violation, because `actions.pure` is not recognized as THE `pure` action creator (which is the only context in which calling other action creators is sensible). I will add "actions.pure" into the list to allow for this usage.

**Comment by VanTanev** [comment]
Thank you!
